samples/webhooks.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// [START dialogflow_cx_webhook]
+
+exports.handleWebhook = (request, response) => {
+  console.log(request);
+  const tag = request.body.fulfillmentInfo.tag;
+  console.log(tag);
+  let jsonResponse = {};
+  if (tag === 'Default Welcome Intent') {
```

Comment:
Feels like we could do a simple string assignment based on the tag, and then create the `jsonResponse` object once using that text?